projects/extension/src/background/ConnectionManager.test.ts

```diff
@@ -195,7 +195,7 @@ describe('Unit tests', () => {
   test('Get networks/chains', () => {
     // With this look the "chain" is removed intentionally as "chain"
     // object cannot be compared with jest 
-    const tmpChains = manager.networks.map(n => (
+    const tmpChains = manager.registeredClients.map((n: any) => (
```

Comment:
This is because you're using `unknown` below.  You should be able to type this properly to avoid all these warnings